src/components/GameItem.js

```diff
@@ -3,23 +3,44 @@ import '../App.css';
 import ItemIcons from '../ItemIcons.js';
 import PropTypes from 'prop-types';
 
+
 class GameItem extends Component {
-  propTypes = {
+constructor(props) {
+  super(props);
+  this.state = {
+    className: ""
+
+  }
+}
+
+  static propTypes = {
     height: PropTypes.number.isRequired,
     layer: PropTypes.number.isRequired,
+    type: PropTypes.string.isRequired,
   }
 
+
+markAsSpotted = () => {
+  if (this.props.type === "litter") {
+    this.setState({className: "spotted-litter"});
+    this.props.itemClicked(this.props.type);
+  } else {
+    this.setState({className: "spotted-nature"});
```

Comment:
Generally I prefer to calculate `className`s for components at the last possible minute.

This is because I consider the `className` and the spotted state to be separate concerns and if you needed to refer to it later (so to avoid double counting) it would be cleaner to check if this component was spotted than if `className` is not empty string.